Lib/_pyrepl/base_eventqueue.py

```diff
@@ -31,13 +31,16 @@
 from .trace import trace
 
 class BaseEventQueue:
+    _ESCAPE_TIMEOUT_MS = 50
```

Comment:
I don't think it's correct to expect timeouts. People can write slowly. Or not. Hold a key, or not. I also can't figure an example of when this would happen.